packages/server/graphql/mutations/helpers/notifications/MSTeamsNotifier.ts

```diff
@@ -162,6 +162,21 @@ export const MSTeamsNotificationHelper: NotificationIntegrationHelper<MSTeamsNot
     const meetingEndedReviewColumn = new AdaptiveCards.Column()
     meetingEndedReviewColumn.width = 'auto'
 
+    if (summary) {
+      const aiSummaryColumnSet = new AdaptiveCards.ColumnSet()
+      const aiSummaryColumn = new AdaptiveCards.Column()
+      aiSummaryColumn.width = 'stretch'
+      const aiSummaryTitle = new AdaptiveCards.TextBlock('AI Summary 🤖')
+      aiSummaryTitle.weight = AdaptiveCards.TextWeight.Bolder
+      const trimmedAISummary = summary.trim()
```

Comment:
+1 It would be nice if we could clean the summary before storing it, so we don't need to remember trimming it each time we're presenting it

Reply:
I like it - I've moved the trim to the OpenAIServerManager 👍